packages/shared/package.json

```diff
@@ -52,6 +52,6 @@
   "dependencies": {
     "glob-to-regexp": "0.4.1",
     "js-cookie": "3.0.1",
-    "swr": "1.3.0"
+    "swr": "2.2.0"
```

Comment:
Out of curiosity - did we face any issues with v1.3.0?

Reply:
Replying to myself: we're using `swr/infinite`

Reply:
Correct, v2 introduces swr/infinite + some nice properties return from useSWR like `isLoading`